docs/faq.rst

```diff
@@ -98,3 +98,15 @@ and users?
 
         key = ProjectKey.objects.filter(project=project)[0]
         print 'SENTRY_DSN = "%s"' % (key.get_dsn(),)
+
+
+.. class:: qa
+
+Is MySQL (or any database besides PostgreSQL) supported?
+
+    Sentry does not support any databases besides PostgreSQL. While, for historical reasons, some
+    things may function, consider using another database equivilent to breaking the factory warranty
+    seal on your new laptop. If you choose to go down that path you'll receive no support from the
```

Comment:
I feel like the tone of this sentence feels a little harsh, something like “Choosing to operate Sentry with an unsupported database will result in the sentry team being unable to provide support, and will result in support tickets and issues posted to the public tracker being closed as ‘wontfix’”.

Reply:
how about:

    Sentry does not support any databases besides PostgreSQL. While, for historical reasons, some
    things may function, choosing to operate Sentry with an unsupported database will result in the
    Sentry team being unable to provide support, and will result in support tickets and issues posted
    to the public tracker being closed as ‘wontfix’

Reply:
🥇